cli_reference/basic_cli_operations.adoc

```diff
@@ -333,6 +333,15 @@ one in the build source. For example, *_pom.xml_* or *_Dockerfile_*.
 ----
 $ oc start-build --from-file=<file_name>
 ----
+Download the binary input using http/https instead of reading it from the file
+system:
```

Comment:
Include an empty line above the code block per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks